R/ctwas_wrapper.R

```diff
@@ -40,16 +40,17 @@
 #' @examples
 #' results <- generate_twas_db(
 #'   weight_db_file = "path/to/weights.rds", conditions = c("Mic", "Oli", "Exc"),
-#'   max_var_selection = 10, min_rsq_threshold = 0.01, p_val_cutoff = 0.05
+#'   min_rsq_threshold = 0.01, p_val_cutoff = 0.05
 #' )
 # select variants for ctwas weights input
 generate_twas_db <- function(weight_db_file, contexts = NULL, variable_name_obj = c("preset_variants_result", "variant_names"),
```

Comment:
@Chunmingl do we already have variant_names inside of the new `*_twas.rds` files? if so it may not be called `c("preset_variants_result", "variant_names")` right? I cannot remember the details but I don't think there is `preset_variants_result` anymore.

Reply:
sorry it was the default names that were not updated, the actual line that were used were written in the sos pipeline which is 
```
rs = generate_twas_db(weight_db_file, contexts=NULL,variable_name_obj=c("variant_names"),susie_obj="susie_weights_intermediate",
                                     twas_weights_table = "twas_weights", min_rsq_threshold = 0.01,
                                     p_val_cutoff = 0.05, data_type_table=xqtl_type_table)
```

Reply:
okay please fix it here.

Reply:
it is now fixed in the updated PR